proto/vtadmin.proto

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
This file was thieved directly from https://github.com/vitessio/vitess/pull/7187/files#diff-eeea62c2adbce1061fd9db0eb38fcb936efb2e158ed0d332bfa5568db563a1c3